core/tests/protractor_utils/waitFor.js

```diff
@@ -61,10 +61,12 @@ var pageToFullyLoad = async function() {
   // https://github.com/angular/angular.js/issues/14219#issuecomment-251605766
   // and browser.waitForAngular's flakiness
   // https://github.com/angular/protractor/issues/2954.
+  await browser.waitForAngularEnabled(false);
   var loadingMessage = element(by.css('.protractor-test-loading-fullpage'));
   await browser.driver.wait(
     until.invisibilityOf(loadingMessage), 15000,
     'Page takes more than 15 secs to load');
+  await browser.waitForAngularEnabled(true);
```

Comment:
Before we call `browser.waitForAngularEnabled(false) ... browser.waitForAngularEnabled(true)`, the e2e tests will automatically wait for pending HTTP requests to finish. After this code, we'll have to use waitFor statements to do that waiting manually. ([source](https://stackoverflow.com/questions/42648077/how-does-waitforangularenabled-work/44073142#44073142)) We currently already have this waitForAngularEnabled code when signing up ([link](https://github.com/oppia/oppia/blob/cd4fbe1d47c4b1befa9fe2391e5d59eff82f0d32/core/tests/protractor_utils/users.js#L90-L92)), so adding this code here will increase flakiness in tests before signup. To fix this, you'll need to add waitFor statements in that pre-signup code